common-ui/src/main/res/values/styles.xml

```diff
@@ -313,7 +309,7 @@
     </style>
 
     <style name="LocationPermissionDialogSplitter">
-        <item name="android:background">?toolbarBgBorderColor</item>
```

Comment:
Should we remove `toolbarBgBorderColor` attr? I checked and we are not using it anymore